app/models/site.rb

```diff
@@ -42,6 +43,12 @@ def needs_geolocation?
     )
   end
 
+  def add_travel_to_work_area_and_london_borough
+    return false unless needs_travel_to_work_area_and_london_borough_updated?
```

Comment:
```suggestion
```

Don't thing you need that now after 

`   after_commit :add_travel_to_work_area_and_london_borough, if: :needs_travel_to_work_area_and_london_borough_updated?`